### PR TITLE
Drop test of IS_WIN, relies on pathlib.PurePath instead

### PR DESCRIPTION
**Pull request by zas**
# Summary

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): PICARD-XXX

# Solution

# Action

### CODE REVIEW

File: picard/util/__init__.py [hunk 1]
```diff
@@ -552,11 +552,11 @@ def album_artist_from_path(filename, album, artist):
         A tuple (album, artist)
     """
     if not album:
-        if IS_WIN:
-            filename = ntpath.splitdrive(filename)[1]
-        dirs = os.path.dirname(filename).replace('\\', '/').lstrip('/').split('/')
+        from pathlib import PurePath
```
Comment (phw) [inline picard/util/__init__.py:555]: Let's put this import to the top

Comment (zas) [reply picard/util/__init__.py]: Done & rebased

**Comment by phw** [comment]
Thanks, that simplifies this even more. We can likely use this in other places as well :)

**Comment by zas** [comment]
> Thanks, that simplifies this even more. We can likely use this in other places as well :)

I'm definitively to use `pathlib` more in Picard code.

**Pull request merged**